lib/torrent.js

```diff
@@ -1063,6 +1063,8 @@ Torrent.prototype._request = function (wire, index, hotswap) {
   var self = this
   var numRequests = wire.requests.length
 
+  var webPeer = wire.remotePort === 80 || wire.remotePort === 443
```

Comment:
Normal peers can listen on port 80 or 443. There are no rules about what ports a client can use, so this might match a normal peer. Not a huge deal right now, but it's better to be explicit since we might add other custom code for web seeds in the future and it could cause trouble.